src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -109,7 +109,7 @@ public int getTotalMessageCount(){
         long timestamp = (long) entity.getProperty("timestamp");
         String recipient = (String) entity.getProperty("recipient");
```

Comment:
Is this line needed here? 
Argument also has the recipient, right?

Reply:
No, it is not, thank you for catching that. I will remove it

Reply:
Actually, oops, it is needed because for that method, getAllMessages(), there aren't any arguments.

Reply:
sorry, my bad!